src/Library/WizardControl.cs

```diff
@@ -495,19 +503,22 @@ private void ConfigureWindowFrame()
 				titleBar.BackColor = commandArea.BackColor;
 			}
 
-			if (parentForm == null) return;
+			if (parentControl == null) return;
 			if (!SuppressParentFormCaptionSync)
-				parentForm.Text = Title;
-			if (!SuppressParentFormIconSync && titleImageIcon != null)
+				parentControl.Text = Title;
+			if (parentAsForm != null)
```

Comment:
`if (parentControl is Form parentAsForm)`

Reply:
Fixed, I accidentally created a new pull request. This one can be rejected.